api/types/types.go

```diff
@@ -87,12 +87,38 @@ type DiskUsageOptions struct {
 // DiskUsage contains response of Engine API:
 // GET "/system/df"
 type DiskUsage struct {
-	LayersSize  int64
-	Images      []*image.Summary
-	Containers  []*container.Summary
-	Volumes     []*volume.Volume
-	BuildCache  []*BuildCache
-	BuilderSize int64 `json:",omitempty"` // Deprecated: deprecated in API 1.38, and no longer used since API 1.40.
```

Comment:
We shouldn't make backwards incompatible changes to public API structs in a minor release.

Reply:
I've reverted this struct back. Is this concern about the Go API type or the HTTP response type? I can also revert the type the HTTP API returns too if that's important.

Reply:
Both really. From the Go API client perspective, vendoring the client update with the struct changed would break existing code (even if it does pin the HTTP version).

On the HTTP API side, unfortunately it's a common practice for the clients to NOT pin to the exact API version, so we might need to have one API version cycle where we deprecate the old response first.